lnst/RecipeCommon/TrafficControl.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import subprocess
+import time
+from tempfile import NamedTemporaryFile
+
+
+class TrafficControlTester:
+    """
+    Class used as a more direct way to run tc to get better stats
+    around it. Can probably be migrated into something like an
+     Iperf.py, Neper.py, etc or something else.
```

Comment:
yeah, this likely makes sense to implement as a subclass of `lnst.Tests.BaseTestModule.BaseTestModule` class.

I'd still probably keep it local to the recipe instead of in the `lnst.Tests` or in the `lnst.RecipeCommon` packages and this should work with `self.host.run(TrafficControlTest(params))` and it will also automatically generate result objects.

This will also take care of running the module in a separate process isolated from the main Agent process which means that it will be more resilient to crashes.